resources/views/accordion.blade.php

```diff
@@ -42,7 +45,7 @@ class="transition duration-150 ease-in-out transform {{ $circleClass }}"
                 </button>
             </dt>
 
-            <dd class="mt-2 @if($isDark) border-l border-theme-secondary-800 pl-8 @endif" x-show.transition.opacity="openPanel" x-cloak>
+            <dd class="mt-2 {{ $slotWrapperClass }} @if($isDark) border-l border-theme-secondary-800 pl-8 @endif" x-show.transition.opacity="openPanel" x-cloak>
```

Comment:
when lines get this long it's usually worth splitting the properties onto separate lines. I tend to just do it when more than 1 property is on the same line. Otherwise you have to scan the whole line to see what properties are used

```suggestion
            <dd
            	class="mt-2 {{ $slotWrapperClass }} @if($isDark) border-l border-theme-secondary-800 pl-8 @endif"
            	x-show.transition.opacity="openPanel"
            	x-cloak
            >
```